exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateTableTests.kt

```diff
@@ -629,4 +630,26 @@ class CreateTableTests : DatabaseTestsBase() {
             }
         }
     }
+
+    @Test
+    fun `create table with dot in name without creating schema beforehand`() {
+        withDb(excludeSettings = listOf(TestDB.ORACLE)) {
```

Comment:
Could you please add a comment here that the Oracle exclusion is not meant to be permanent (or is excluded because of expected behavior)? Something so we know that the test is supposed to pass and it can be included once a fix for the casing issue is merged.

Reply:
It would be great also to log an issue to fix it in the future